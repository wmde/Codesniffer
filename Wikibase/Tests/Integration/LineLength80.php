<?php

// phpcs:property PHP_CodeSniffer\Standards\Generic\Sniffs\Files\LineLengthSniff::$absoluteLineLimit = 80
