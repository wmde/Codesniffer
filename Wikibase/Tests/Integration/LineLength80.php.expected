 3 | ERROR | Line exceeds maximum limit of 80 characters; contains 105 characters
